graphql_compiler/tests/test_sqlalchemy_schema_generation.py

```diff
@@ -55,3 +61,37 @@ def test_sql_self_inheritance(self):
         # Tests that a class is in its own subclass/superclass set.
         self.assertEqual(self.schema_graph.get_subclass_set('A'), {'A'})
         self.assertEqual(self.schema_graph.get_superclass_set('A'), {'A'})
+
+    def test_sqlalchemy_graphql_schema_generation(self):
```

Comment:
Can we add the ordering of directives, types, etc to the spec of the function?

Reply:
I'll do this but in a different branch and then I'll merge it onto this one.

Reply:
Right now the generated string is always deterministically the same given two equivalent SchemaGraph objects. However, the ordering specification is quite complicated right now. In another PR I will simplify the ordering and write the specification.

Reply:
If you look at it right now, it seems that the directives are listed at the top and the rest is alphabetical. This is not always the case. The perfectly alphabetical ordering is a bit of a coincidence.